packages/pds-ember/app/styles/pds/core/typography/inline-code/_config.scss

```diff
@@ -0,0 +1,41 @@
+$_module: "PDS.Core.Typography.InlineCode";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "../../../theme";
+
+$backgroundColor: theme.$inlineCode-backgroundColor;
+$borderColor: theme.$inlineCode-borderColor;
+$borderRadius: theme.$inlineCode-borderRadius;
+$color: theme.$inlineCode-color;
+$font: theme.$inlineCode-font;
+$fontFamily: theme.$inlineCode-fontFamily;
+$lineHeight: theme.$inlineCode-lineHeight;
+$padding: theme.$inlineCode-padding;
```

Comment:
Interesting - are you using these vary anywhere external to this module?

Reply:
Or is this so you could override the theme styles for this specific component?

Reply:
Not currently. I'm experimenting with adding component-specific vars to the theme file for consolidation of themable properties.